requirements.txt

```diff
@@ -0,0 +1,104 @@
+asn1crypto==0.22.0
+Babel==2.3.4
+Beaker==1.8.1
+beautifulsoup4==4.5.1
+bleach==1.5.0
+blinker==1.4
+certifi==2017.7.27.1
+cffi==1.11.0
+-e git+https://github.com/andela/ckan.git@c8b60734e89b402f9f9f04a2daf0be19d13164ae#egg=ckan
+-e git+https://github.com/andela/ckanext-datarequests.git@1d08974a35cb25091a07ea3da3768fe0a505b02c#egg=ckanext_datarequests
+-e git+https://github.com/andela/ckanext-followy.git@5acc96ea5770f2c847e32b8bbefad27e3886dc69#egg=ckanext_followy
+-e git+https://github.com/okfn/ckanext-harvest.git@aca5b42b5635e03b96da1b3bc788c687031e6f38#egg=ckanext_harvest
+-e git+https://github.com/andela/ckanext-openafrica.git@753191fcb417797d9b447369acdddb84a086eace#egg=ckanext_openafrica
+click==6.7
+coverage==4.4.1
+coveralls==1.2.0
+cryptography==2.0.3
+decorator==4.0.6
+docopt==0.6.2
+docutils==0.12
+enum34==1.1.6
+extras==1.0.0
+factory-boy==2.1.1
+fanstatic==0.12
+first==2.0.1
+fixtures==3.0.0
+Flask==0.11.1
+Flask-Babel==0.11.2
+Flask-DebugToolbar==0.10.0
+Flask-Moment==0.5.1
+Flask-Testing==0.6.2
+FormEncode==1.3.0
+funcsigs==1.0.2
+html5lib==0.9999999
+httpretty==0.8.3
+idna==2.6
+ipaddress==1.0.18
+itsdangerous==0.24
+Jinja2==2.8
+libmagic==1.0
+linecache2==1.0.0
+Mako==1.0.4
+Markdown==2.6.7
+MarkupSafe==0.23
+mock==2.0.0
+mox3==0.23.0
+nose==1.3.7
+ofs==0.4.2
+ordereddict==1.1
+Pairtree===0.7.1-T
+passlib==1.6.5
+Paste==1.7.5.1
+PasteDeploy==1.5.2
+PasteScript==2.0.2
+pbr==1.10.0
+pika==0.9.8
+pip-tools==1.7.0
+polib==1.0.7
+psycopg2==2.4.5
+py==1.4.34
+pycodestyle==2.2.0
+pycparser==2.18
+pyfakefs==2.9
+Pygments==2.1.3
+Pylons==0.9.7
+pyOpenSSL==17.3.0
+pysolr==3.6.0
+pytest==3.2.2
+python-dateutil==1.5
+python-geoip==1.2
+python-geoip-geolite2==2015.303
+python-magic==0.4.12
+python-mimeparse==1.6.0
+pytz==2016.7
+pyutilib.component.core==4.6.4
+redis==2.10.5
+repoze.lru==0.6
+repoze.who==2.3
+repoze.who-friendlyform==1.0.8
+requests==2.11.1
+Routes==1.13
+rq==0.6.0
+selenium==3.6.0
+simplejson==3.10.0
+six==1.10.0
+Sphinx==1.2.3
+sphinx-rtd-theme==0.1.9
+SQLAlchemy==1.1.11
+sqlalchemy-migrate==0.10.0
+sqlparse==0.2.2
+Tempita==0.5.2
+testtools==2.3.0
+traceback2==1.4.0
+tzlocal==1.3
+unicodecsv==0.14.1
+unittest2==1.1.0
+urllib3==1.22
+vdm==0.14
+WebError==0.13.1
+WebHelpers==1.3
+WebOb==1.0.8
+WebTest==1.4.3
+Werkzeug==0.11.10
+zope.interface==4.3.2
```

Comment:
These requirements need to be reduced to only the ones not already in CKAN.

Reply:
I don't quite understand what you mean here

Reply:
Remove all the packages/modules that are already listed in CKAN.

Reply:
Done. Changes are visible on the test-forward-compatibility PR.